.github/workflows/ci_docs.yml

```diff
@@ -1,45 +1,127 @@
 name: Nim Docs CI
 on:
+  push:
+    # Run only on changes on these files
+    paths:
+      - 'lib/**.nim'
+      - 'doc/**.rst'
+      - 'doc/nimdoc.css'
```

Comment:
@alaviss what about `tools/dochack` which includes `tools/dochack/dochack.nim`?

Reply:
Feel free to PR that in.